src/main/kotlin/value/VFun.kt

```diff
@@ -35,20 +37,19 @@ data class VFun(
     }
 
     fun verbInvoke(c: Context, args: List<Value>): Value {
-        getVerb()?.also { verb ->
-            if ((args.size < argNames.size) || (args.size > argNames.size && args.size > 1))
-                fail(E_RANGE, "lambda wants ${argNames.size} args but got ${args.size}")
-
-            return verb.call(c, vThis, listOf(), entryPoint, withVars = buildMap {
-                vars.forEach { (name, v) -> put(name, v) }
-                argNames.forEachIndexed { n, name ->
-                    args.getOrNull(n)?.also { put(name, it) } ?: fail(E_RANGE, "arg $name not found")
-                }
-                if (argNames.isEmpty() && args.size == 1) put("it", args[0])
-            })
-
-        } ?: fail(VMException.Type.E_SYS, "missing verb for VFun ($traitID.$name)")
-        return VVoid
+        if ((args.size < argNames.size) || (args.size > argNames.size && args.size > 1))
+            fail(E_RANGE, "lambda wants ${argNames.size} args but got ${args.size}")
+        val vm = VM(this)
```

Comment:
This code is now pretty similar to Verb.call(); these could come closer together in the future...